exporter/downloadscollector.go

```diff
@@ -60,6 +62,14 @@ func NewDownloadsCollector(ds DownloadsSource) *DownloadsCollector {
 		labels = []string{"info_hash", "name"}
 	)
 
+	// namespace is the top-level namespace for this rTorrent exporter.
```

Comment:
I don't understand the need for this change.  Can you please clarify?  I am not aware of any other exporters that allow modifying the top-level namespace.